rust/meson.build

```diff
@@ -0,0 +1,25 @@
+cargo_cmd = [cargo, 'build', '--manifest-path=@INPUT@', '--target-dir=@OUTDIR@',
+             cargo_build_args]
+
+# the target to compile all rust binaries
+custom_target('rust_binaries',
+              output: '@PLAINNAME@.__PHONY__',
+              input: 'Cargo.toml',
+              command: cargo_cmd,
+              env: cargo_env,
+              depends: sched_deps,
+              build_by_default: true,
+              build_always_stale: true)
```

Comment:
I would make it explicitly build only `rust_binaries`. No need to build libraries separately by default.

Reply:
do you mean to set `build_always_stale` to false on `rust_binaries`?

Reply:
There need to be two targets:
1. The default target which gets triggered by default.
2. Package-specific target so that users can do `meson compile -C build $TARGET`.
We don't want to build libraries by default for 1) but want to provide (we weren't before) individual targets for 2).

So, the default target should do `cargo build -p scx_loader`. It will probably look something like:
```
rust_libs = [ ... ]
rust_bins = [ ... ]
rust_all = rust_libs + rust_bins

// `-p bin` for each in rust_bin to cargo args
custom_target('rust_binaries', ...)

foreach package in rust_all
    custom_target(package, ...)
```

Please feel free to leave it as is. I can update afterwards too.

Reply:
Oh. I'll leave it as is then )